events-api/src/modules/events/events.service.ts

```diff
@@ -61,8 +53,73 @@ export class EventsService {
       },
       skip: (page - 1) * limit,
       take: limit,
+      orderBy: { [sortBy]: sortOrder },
     });
 
     return events;
   }
+
+  async updateEvent(eventId: string, dto: UpdateEventReqDto) {
+    await this.verifyEventExistence(eventId);
```

Comment:
Why to have this method? I'd say just get the event by id and explicitly throw an error. I'd advice you to be explicit about your code, rather being too implicit.

Reply:
> Why to have this method? I'd say just get the event by id and explicitly throw an error. I'd advice you to be explicit about your code, rather being too implicit.

fixed